test/Engine/ProtoTest/Associative/Inheritance.cs

```diff
@@ -8,7 +8,7 @@ namespace ProtoTest.Associative
     class InheritaceTest : ProtoTestBase
     {
         [Test]
-        [Category("DSDefinedClass")]
+        [Category("DSDefinedClass_Ignored_DoNotTestInheritance")]

```

Comment:
The intention is to not ignore them at this point.
 At the end of the test porting exercise, I intended to do a global replace of DSDefinedClass_Ignored with an actual Ignore category or deleting them altogether